task_slack_posts.py

```diff
@@ -5,12 +5,12 @@
 """
 from tulflow import tasks
 
-def slackpostonsuccess(dag, **context):
+def slackpostonsuccess(**context):
```

Comment:
this is `**context`, but just below, slackpostonfail uses  just plain`context`. Is this difference intentional?

Reply:
Yes.